phpBB/includes/functions_compatibility.php

```diff
@@ -943,3 +943,76 @@ function parse_cfg_file($filename, $lines = false)
 
 	return $parsed_items;
 }
+
+/**
+* Wraps an url into a simple html page. Used to display attachments in IE.
```

Comment:
I don't think we need this anymore as IE shouldn't be supported anymore.

Reply:
The same, this file will be deleted in phpbb4, so there is no problem moving it here

Reply:
Ok, fine for now.